assets/stories/skate-components/map/markers/locationDotMarker.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import React from "react"
+import { LocationMarker } from "../../../../src/components/mapMarkers"
+import { inMapDecorator } from "../../../../.storybook/inMapDecorator"
+
+const location: LocationSearchResult = {
+  id: "10pp",
+  name: "MBTA Offices",
+  address: "10 Park Plaza",
+  latitude: 42.360082,
+  longitude: -71.05888,
+}
+
+const meta = {
+  component: ({ selected }) => {
+    return <LocationMarker location={location} selected={selected} />
+  },
```

Comment:
question: [is there benefit/what happens] if you pass the `LocationMarker` directly?
```suggestion
  component: LocationMarker,
  args: {
    location
  },
```